### PR TITLE
Improve tag tips in product page

### PR DESCRIPTION
**Pull request by vincentbz**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | I complete the tips message for tags in the BO product page, saying it needs to rebuild the index.
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-1127
| How to test?  | Just see the text, it should talk about the index

### CODE REVIEW

File: src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig [hunk 1]
```diff
@@ -783,7 +783,7 @@
                         <div class="alert alert-info alert-down" role="alert">
                           <p class="alert-down-text">
                             {{ 'Choose terms and keywords that your customers will use to search for this product and make sure you are consistent with the tags you may have already used.'|trans({}, 'Admin.Catalog.Help')|raw }}<br>
-                            {{ 'You can manage tag aliases in the [1]Search section[/1].'|trans({}, 'Admin.Catalog.Help')|
+                            {{ ' If you add new tags, you have to rebuild the index. You can manage tag aliases in the [1]Search section[/1].'|trans({}, 'Admin.Catalog.Help')|
```
Comment (maximebiloe) [inline src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig]: I think there is an extra space 😉

**Review by maximebiloe — DISMISSED** [review]

File: src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig (hunk 1 above)
Comment (Quetzacoalt91) [inline src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig]: Space removed!

**Comment by AlexEven** [comment]
hello, 

I think the sentence should be the last one. The index is in the search page that needs to be introduced first.
And "add new tags" it means creating new tags or adding more tags to an already existing products with tags? Just to be sure :)

Thanks!

**Comment by aleeks** [comment]
Thank you @vincentbz

**Comment by aleeks** [comment]
Can we have this ? "You can manage tag aliases in the [1]Search section[/1]. If you add new tags, you have to rebuild the index." ??????

File: src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig [hunk 2]
```diff
@@ -783,7 +783,7 @@
                         <div class="alert alert-info alert-down" role="alert">
                           <p class="alert-down-text">
                             {{ 'Choose terms and keywords that your customers will use to search for this product and make sure you are consistent with the tags you may have already used.'|trans({}, 'Admin.Catalog.Help')|raw }}<br>
-                            {{ 'You can manage tag aliases in the [1]Search section[/1].'|trans({}, 'Admin.Catalog.Help')|
+                            {{ 'If you add new tags, you have to rebuild the index. You can manage tag aliases in the [1]Search section[/1].'|trans({}, 'Admin.Catalog.Help')|
```
Comment (aleeks) [inline src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig]: You can manage tag aliases in the [1]Search section[/1]. If you add new tags, you have to rebuild the index.

**Review by aleeks — DISMISSED** [review]

File: src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig (hunk 2 above)
Comment (Quetzacoalt91) [inline src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig]: I commited for @vincentbz to go faster.

**Comment by aleeks** [comment]
Thank you @vincentbz  & @Quetzacoalt91 !

**Pull request merged**